infrastructure/api/api_client.go

```diff
@@ -0,0 +1,64 @@
+package api
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/wmaldonadoc/academy-go-q42021/constants"
+	"github.com/wmaldonadoc/academy-go-q42021/infrastructure/exceptions"
+
+	"go.uber.org/zap"
+)
+
+/*
+	ApiResponse - Represents an abstraction of HTTP response.
+	- Headers: HTTP response headers
+	- Body: HTTP response body as string
+	- HTTPStatus: HTTP response status
+*/
+type ApiResponse struct {
```

Comment:
For better readability, all acronyms should be written with uppercase letters. Please fix all occurrences of this issue.

```suggestion
type APIResponse struct {
```